demos/demos_by_use_case/bio/MetagenomicDemo.ipynb

```diff
@@ -0,0 +1,1975 @@
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "data=pd.read_csv('/content/PRJNA544527_mpa4out.txt',sep='\\t',skiprows=1,index_col=0)\n",
+        "\n",
+        "g = graphistry.nodes(cudf.from_pandas(data.dropna()))\n",
+        "\n",
+        "t=time()\n",
+        "g3=g.umap(dbscan=True,engine='umap_learn')\n",
+        "print('\\n Total ', np.round(time() - t,1), 'seconds passed')\n",
+        "\n",
+        "emb2=g3._node_embedding\n",
+        "g22=graphistry.nodes(emb2.reset_index(),'index').edges(g3._edges,'_src_implicit','_dst_implicit').bind(point_x=\"x\",point_y=\"y\").settings(url_params={\"play\":0})"
+      ],
+      "metadata": {
+        "id": "HrJErY1SZSfh",
+        "outputId": "a5e6cd2b-e198-460e-ca02-46bb6cc592a6",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        }
+      },
+      "execution_count": 38,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "WARNING:graphistry.umap_utils:* Ignoring target column of shape (679, 0) in UMAP fit, as it is not one dimensional"
+          ]
+        },
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "\n",
+            " Total  24.8 seconds passed\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## UMAP by species via GPU"
+      ],
+      "metadata": {
+        "id": "l7aLIxP6ZY9s"
+      }
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 28,
+      "metadata": {
+        "id": "6Zoo17Ui9zxh",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "6af64cb4-1528-4033-f606-75fb73af4686"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "WARNING:graphistry.util.memoization:! Failed umap speedup attempt. Continuing without memoization speedups.WARNING:graphistry.umap_utils:* Ignoring target column of shape (679, 0) in UMAP fit, as it is not one dimensional"
+          ]
+        },
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "\n",
+            " Total  0.7 seconds passed\n"
+          ]
+        }
+      ],
+      "source": [
+        "data=pd.read_csv('/content/PRJNA544527_mpa4out.txt',sep='\\t',skiprows=1,index_col=0)\n",
+        "\n",
+        "g = graphistry.nodes(cudf.from_pandas(data.dropna()))\n",
+        "\n",
+        "t=time()\n",
+        "g3=g.umap(dbscan=True,engine='cuml')\n",
+        "print('\\n Total ', np.round(time() - t,1), 'seconds passed')\n",
+        "\n",
+        "emb2=g3._node_embedding\n",
+        "g22=graphistry.nodes(emb2.reset_index(),'index').edges(g3._edges,'_src_implicit','_dst_implicit').bind(point_x=\"x\",point_y=\"y\").settings(url_params={\"play\":0})"
```

Comment:
afaict in both of these demos we can go straight to `g.plot()` instead of computing `g22`